docs/docs/migration-guide.mdx

```diff
@@ -13,6 +13,29 @@ This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
 
+## Rasa 3.2 to 3.3
+
+### ConcurrentRedisLockStore
```

Comment:
What about migration of the DB? Do users need to specify a different DB number? What happens to previous locks?
These are questions users will probably have

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/11342/commits/4df9022f02a2697dfa56f03a5ccda4810369ff47